agent/global.go

```diff
@@ -35,4 +35,8 @@ const (
 	// maxHeartbeats is the number of failed heartbeats after which an agent
 	// reports an error
 	maxHeartbeats = 10
+
+	// setupTimeout is how long the agent will wait before timing out running
+	// the setup script.
+	setupTimeout = 2 * time.Minute
```

Comment:
this could be longer since it's no longer in a critical path?

Reply:
Is there a risk from the monitor? Do ever expect a timeout script to take two minutes? I almost reduced it.